middleware/error_handler.go

```diff
@@ -38,7 +40,7 @@ func ErrorHandler(service *goa.Service, verbose bool) goa.Middleware {
 					reqID = shortID()
 					ctx = context.WithValue(ctx, reqIDKey, reqID)
 				}
-				goa.LogError(ctx, "uncaught error", "id", reqID, "msg", respBody)
+				goa.LogError(ctx, fmt.Sprintf("uncaught error : %+v", e), "id", reqID, "msg", respBody)
```

Comment:
This seems reasonable, I think it is just a bit of unfamiliarity. Will update PR